src/style.css

```
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+}
+
+body ,
+h2 ,
+p {
+  margin: 0;
+}
+
```

Я б не стилізував теги. Марджини можна скинути в *.
```suggestion
* {
  box-sizing: border-box;
  margin: 0;
}
```

---

@SvyatoslavYatsenko don’t change margins with * selector